src/test/java/BaseTest.java

```diff
@@ -1,18 +1,111 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.firefox.FirefoxDriver;
-import org.openqa.selenium.safari.SafariDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Parameters;
 
+import java.time.Duration;
 
-public class BaseTest {
 
+public class BaseTest {
+   public static WebDriver driver = null;
+   public static String url = null;
+   public static WebDriverWait wait = null;
+  
     @BeforeSuite
     static void setupClass() {
         WebDriverManager.chromedriver().setup();
     }
+    @BeforeMethod
+    @Parameters({"baseUrl"})
+    public void launchBrowser(String baseUrl){
+        driver = new ChromeDriver();
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+        driver.manage().window().maximize();
+        url = baseUrl;
+        driver.get("url");
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/224928887-94d0f72b-0e40-4204-95f6-468cef415bba.png)

**HINT**

![image](https://user-images.githubusercontent.com/104761482/224929045-199c0979-7a53-4547-8fe4-81bcc4ed37d0.png)